mongo/tox.ini

```diff
@@ -3,15 +3,14 @@ isolated_build = true
 minversion = 2.0
 basepython = py38
 envlist =
-    py38-{4.4,5.0}-{standalone,shard,tls}
+    py38-{4.4,5.0}-{standalone,shard,auth,tls}
 
 [testenv]
 ensure_default_envdir = true
 envdir =
-    py27: {toxworkdir}/py27
```

Comment:
With this new version of pymongo, are we dropping support for Python 2? We probably should label this as `changelog/Changed` in that case? @hithwen

Reply:
yes, we should

Reply:
Done